server/platform/services/sharedchannel/service.go

```diff
@@ -261,7 +261,6 @@ func (scs *Service) notifyClientsForSharedChannelConverted(channel *model.Channe
 }
 
 func (scs *Service) notifyClientsForSharedChannelUpdate(channel *model.Channel) {
-	scs.platform.InvalidateCacheForChannel(channel)
```

Comment:
Hey @wiggin77 - I was looking at the cache invalidation code and this part looked a bit strange to me. From the caller, you are only updating the `SharedChannels` table. There is no change happening to the `Channels` table. Do we still need to invalidate the channel cache here? Or is there a bug where we have missed out on actually updating the channels table?

Reply:
I need to look deeper into whether a WS event is needed here at all.